google-cloud-bigquery/pom.xml

```diff
@@ -51,7 +51,6 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
       <scope>test</scope>
```

Comment:
I'm not a maven expert so it would help to get some explanation - what is the difference between not having this dependency in this file, and having it here with no version?